frontend/packages/dev-console/src/components/pipelines/__tests__/PipelineDetailsPage.spec.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { referenceForModel } from '@console/internal/module/k8s';
+import { DetailsPageProps } from '@console/internal/components/factory';
+import { PipelineModel } from '../../../models';
+import PipelineDetailsPage from '../PipelineDetailsPage';
+
+const pipelineDetailsProps: DetailsPageProps = {
```

Comment:
can use:
```suggestion
const pipelineDetailsProps: React.ComponentProps<typeof PipelineDetailsPage> = {
```
and remove the `DetailsPageProps` import

Reply:
done